influxdb_client/client/write/point.py

```diff
@@ -195,6 +195,24 @@ def write_precision(self):
         """Get precision."""
         return self._write_precision
 
+    @classmethod
+    def set_str_rep(cls, rep_function):
+        """Set the string representation for all Points."""
+        cls.__str___rep = rep_function
+
+    def __str__(self):
+        """
+        Create string representation of this Point.
+
+        Can be set via `Point.set_str_rep`. Defaults to `to_line_protocol`
+        Example:
+            .. code-block:: python
+                Point.set_str_rep(lambda p: f'{p._name} - {p._tags} - {p._fields} - {p._time}')
+        """
+        if self.__str___rep is None:
+            return self.to_line_protocol()
+        return self.__str___rep()
```

Comment:
The `set_str_rep` function is not needed please remove it.

```suggestion
        """
        return self.to_line_protocol()
```